README.md

```diff
@@ -69,7 +69,7 @@ For protobuf-based codegen integrated with the Maven build system, you can use
           protobuf-java directly, you will be transitively depending on the
           protobuf-java version that grpc depends on.
         -->
-        <protocArtifact>com.google.protobuf:protoc:3.0.0-alpha-3.1:exe:${os.detected.classifier}</protocArtifact>
+        <protocArtifact>com.google.protobuf:protoc:3.0.0-beta-1:exe:${os.detected.classifier}</protocArtifact>
```

Comment:
Agreed. We don't change it now.

Right it's not safe to do so. The problem we had earlier was that the dependency on grpc introduced a transitive dependency on a certain version of protobuf runtime, which was different from the version of `protocArtifact`.